src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTests.cs

```diff
@@ -59,20 +59,20 @@ public void ValidateBasicObjectsRoundtrip(object obj, FormatterAssemblyStyle ass
         }
 
         // Used for updating blobs in BinaryFormatterTestData.cs
-        //[Fact]
-        public void UpdateBlobs()
-        {
-            string testDataFilePath = GetTestDataFilePath();
-            string[] coreTypeBlobs = SerializableEqualityComparers_MemberData()
-                .Concat(SerializableObjects_MemberData())
-                .Select(record => BinaryFormatterHelpers.ToBase64String(record[0]))
-                .ToArray();
-
-            var (numberOfBlobs, numberOfFoundBlobs, numberOfUpdatedBlobs) = UpdateCoreTypeBlobs(testDataFilePath, coreTypeBlobs);
-            Console.WriteLine($"{numberOfBlobs} existing blobs" +
-                $"{Environment.NewLine}{numberOfFoundBlobs} found blobs with regex search" +
-                $"{Environment.NewLine}{numberOfUpdatedBlobs} updated blobs with regex replace");
-        }
+        // [Fact]
+        // public void UpdateBlobs()
+        // {
+        //     string testDataFilePath = GetTestDataFilePath();
+        //     string[] coreTypeBlobs = SerializableEqualityComparers_MemberData()
+        //         .Concat(SerializableObjects_MemberData())
+        //         .Select(record => BinaryFormatterHelpers.ToBase64String(record[0]))
+        //         .ToArray();
+
+        //     var (numberOfBlobs, numberOfFoundBlobs, numberOfUpdatedBlobs) = UpdateCoreTypeBlobs(testDataFilePath, coreTypeBlobs);
+        //     Console.WriteLine($"{numberOfBlobs} existing blobs" +
+        //         $"{Environment.NewLine}{numberOfFoundBlobs} found blobs with regex search" +
+        //         $"{Environment.NewLine}{numberOfUpdatedBlobs} updated blobs with regex replace");
+        // }
```

Comment:
I don't like commenting these out.  I'd rather see it made into a [ConditionalFact] that checks for a particular environment variable.  Then to update the blobs, all one needs to do is set that environment variable and run the tests.

Reply:
This helper method doesn't work anymore, therefore I'm deleting it.